examples/tts/conf/de/fastpitch_align_44100.yaml

```diff
@@ -90,7 +90,10 @@ model:
       max_duration: 15  # change to null to include longer audios.
       min_duration: 0.1
       ignore_file: null
-      trim: false
+      trim: true
```

Comment:
Are we sure we want to have this as a default configuration? I would think we should have the default trimming off, but let people know they can turn it on based on their particular dataset.

Reply:
since both @rlangman and @titu1994 agreed to assign `false` by default. I will revise it in next version. I will add params in the description.